CHANGELOG.rst

```diff
@@ -101,6 +101,90 @@ Added
 
 Changed
 ^^^^^^^
+* `@senwu`_: Switch to Emmental as the default learning engine.
+
+.. note::
+
+    Instead of owning Fonduer learning engine, we switch to Emmental which is a deep learning
+    framework for multi-task learning. This change can give Fonduer more power to support more
+    applications. Example usage:
+
+    .. code:: python
+
+        import emmental
+
+        # Collect word counter from candidates
+        word_counter = collect_word_counter(train_cands)
+
+        emmental.init(fonduer.Meta.log_path)
+
+        # Geneate special tokens
```

Comment:
I personally think it would be nice to give more detailed comments here unless you think a user will understand this. Why are these special tokens needed? Why is it in the format in line 125? 

```suggestion
        # Generate special tokens
```